omnigibson/transition_rules.py

```diff
@@ -1104,6 +1140,9 @@ def _compute_container_info(cls, object_candidates, container, global_info):
         in_volume = container.states[ContainedParticles].check_in_volume(obj_positions) | \
                     np.array([obj.states[OnTop].get_value(container) for obj in cls._OBJECTS])
 
+        # Container itself is never within its own volume
+        in_volume[cls._OBJECTS_TO_IDX[container]] = False
+
```

Comment:
I understand the rationale, but why do we need to add a special case line here? What issue is this line trying to solve?